src/runtime/Function.h

```diff
@@ -44,7 +44,11 @@ class Function : public Extern {
 
     const FunctionType* functionType() const { return m_functionType; }
 
-    virtual void call(ExecutionState& state, const uint32_t argc, Value* argv, Value* result) = 0;
+    // client must override one of call function
+    virtual void call(ExecutionState& state, Value* argv, Value* result);
+    virtual void intepreterCall(ExecutionState& state, uint8_t* bp, ByteCodeStackOffset* offsets,
```

Comment:
A minor typo, `interpreterCall`

Reply:
And its a little bit confusing because `ImportedFunction` overrides `call` function while `DefinedFunction` overrides `interpreterCall` function.
Instead, what about just defining both `call` and `interpreterCall` methods as pure virtual functions in `Function` class and overriding each method in subclasses?

Reply:
I consider that decedent of Function inherit one of `call` or `interpreterCall` what developer wanted to.
but there is only two decedent. so I changed implementation as you said